### ISSUE TITLE
Implement regularization dropout

### ISSUE THREAD

**Issue by JT4v4res** [JT4v4res/Neural-networks-and-deep-learning#23]
Dropout technique is one of most used regularization methods in neural networks training.
